src/subgraph/actions/listAllSales.ts

```diff
@@ -17,7 +17,7 @@ export const listAllSales = async <T>(
     const queryResult = await apolloClient.query<TokenSalesDto>({
       query: queries.getAllTokenSales,
       variables: {
-        contract,
+        contract: contract.toLowerCase(),
```

Comment:
remind me why we do this?

Reply:
We removed `toLowerCase` from the API because it was case sensitive and would miss bids that were totally valid. We are also adding here because it's case sensitive, it's just expecting lower case values it seems. 

These queries return empty without doing a `toLowerCase()`